CMakeLists.txt

```diff
@@ -22,7 +22,30 @@ set_env( MOBILEID_URL "https://dd-mid.ria.ee/mid-api" CACHE STRING "URL for Mobi
 set_env( SMARTID_URL "https://dd-sid.ria.ee/v1" CACHE STRING "URL for Smart-ID" )
 set(CMAKE_INTERPROCEDURAL_OPTIMIZATION YES)
 set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_DEBUG NO)
-
+set(CPACK_PACKAGE_VERSION ${VERSION})
+set(CPACK_GENERATOR "RPM")
+set(CPACK_PACKAGE_NAME "qdigidoc4")
+set(CPACK_PACKAGE_RELEASE 1)
+set(CPACK_PACKAGE_CONTACT "Andrus Randveer")
+set(CPACK_PACKAGE_VENDOR "RIA")
+set(CPACK_PACKAGING_INSTALL_PREFIX "/usr/")
+set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION
+        "/usr/local" "/usr/local/etc" "/usr/share/man"
+        "/usr/share/man/man1" "/usr/lib64/pkgconfig" "/usr/share/icons/hicolor"
+        "/usr/share/applications" "/usr/share/icons" "/usr/share/mime" "/usr/share/mime/packages"
+        "/usr/share/icons/hicolor/16x16" "/usr/share/icons/hicolor/16x16/apps" "/usr/share/icons/hicolor/16x16/mimetypes"
```

Comment:
This seems like an anti-pattern. I'm not sure how to avoid it. Without this, there are a bunch of install conflicts.

Reply:
Im not sure what is installed under /usr/local.